bin/parse_dmc_classification.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+import sys
+
+if " " in sys.argv[1] :
+  seqName = sys.argv[1].split(" ")[0]
+else :
+  seqName = sys.argv[1]
+
+if float(sys.argv[2]) != 0 and float(sys.argv[3]) != 0 and float(sys.argv[4]) != 0 and float(sys.argv[5]) != 0 and float(sys.argv[6]) != 0 : 
+  print(f'{seqName}\t{["Eukaryote", "EukaryoteVirus", "Plasmid", "Prokaryote", "ProkaryoteVirus"][[float(sys.argv[2]), float(sys.argv[3]), float(sys.argv[4]), float(sys.argv[5]), float(sys.argv[6])].index(max(float(sys.argv[2]), float(sys.argv[3]), float(sys.argv[4]), float(sys.argv[5]), float(sys.argv[6])))]}')
```

Comment:
```suggestion
# select DMC classification with maximum ratio
if all(float(sys.argv[i]) != 0 for i in range(2, 7)):
  dmc_values = [float(arg) for arg in sys.argv[2:7]]
  max_index = dmc_values.index(max(dmc_values))
  labels = ["Eukaryote", "EukaryoteVirus", "Plasmid", "Prokaryote", "ProkaryoteVirus"]
  print(f'{seq_name}\t{labels[max_index]}')
```
Suggestion to make it a little readable for someone new. Can I also check the logic here. Is it possible that one of the values could be 0 but it will still pass? In which case you just need to find the max without the first if statement?

Reply:
That's also definitively cleaner like that thanks.
I only saw 0.0 in the case where the contig wasn't classified (too many N, length<500pb) by the model, 0.0 for all classes. I think (but I will re-check) the score is everytime different from 0.0 when the model makes prediction, but I wrote this if statement to avoid an unfortunate statistics.